WebApplication/Migrations/20201123222800_DbBasexx.Designer.cs

```diff
@@ -0,0 +1,155 @@
+﻿// <auto-generated />
```

Comment:
Migration classes and the whole folder can be deleted once the migrations are done